web-app/preprocess/classification.py

```diff
@@ -0,0 +1,91 @@
+import sys
+import nltk
+import string
+from nltk.corpus import stopwords
+from nltk.stem import WordNetLemmatizer
+import os
+import pickle
+import numpy as np
+import fasttext.util
+from scipy.spatial.distance import cosine
+# Remove "other" from list of stopwords in order to classify crime types such as "other crime", "etc"
+
+
+def simplifyText(crimeType):
```

Comment:
nit: for Python code, I think the style guide suggest adding docstrings and type annotations.